va_explorer/va_data_management/utils/kobo.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pandas as pd
+import requests
+from requests.auth import HTTPBasicAuth
+
+KOBO_HOST = os.environ.get("KOBO_HOST", "http://127.0.0.1:5003")
+# Don't verify localhost (self-signed cert)
+SSL_VERIFY = os.environ.get(
+    "KOBO_SSL_VERIFY", not KOBO_HOST.startswith("http://localhost")
```

Comment:
IF KOBO_SSL_VERIFY is set in .env, it appears to read in here as a string type "True" or "False" instead of Boolean, which then causes everything to fail.  NOT having it set in .env works just fine.

Proposal for consideration:  at line 12, add: SSL_VERIFY = bool(SSL_VERIFY)

NOTE: I *think* this issue may also appear in odk.py